webapp/channels/src/components/admin_console/team_channel_settings/group/index.js

```diff
@@ -1,30 +0,0 @@
-// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
-// See LICENSE.txt for license information.
-
-import {connect} from 'react-redux';
-
-import {t} from 'utils/i18n';
-
-import List from './group_list';
-
-function mapStateToProps(state, {groups, totalGroups, isModeSync, onGroupRemoved, setNewGroupRole}) {
-    return {
-        data: groups,
-        removeGroup: onGroupRemoved,
-        setNewGroupRole,
-        emptyListTextId: isModeSync ? t('admin.team_channel_settings.group_list.no-synced-groups') : t('admin.team_channel_settings.group_list.no-groups'),
-        emptyListTextDefaultMessage: isModeSync ? 'At least one group must be specified' : 'No groups specified yet',
-        total: totalGroups,
-    };
-}
-
-function mapDispatchToProps() {
-    return {
-        actions: {
-            getData: () => Promise.resolve(),
-        },
-    };
-}
```

Comment:
[3]

Reply:
I am not sure what you meant by the [n] comments. Are those potential problem points within the code commits that may be requiring team_groups and channel_groups to pass an action that returns Promise<Group[]>? I am really lost at this point :(

Reply:
sorry i just used this a reference points to explain in https://github.com/mattermost/mattermost/pull/23511#discussion_r1245506853
apologies for causing this as confusion. But you are doing really great job 💯